sdk/dotnet/Cognito/IdentityPoolRoleAttachment.cs

```diff
@@ -83,7 +83,7 @@ namespace Pulumi.Aws.Cognito
     ///         AssumeRolePolicy = authenticatedPolicyDocument.Apply(getPolicyDocumentResult =&gt; getPolicyDocumentResult.Json),
     ///     });
     /// 
-    ///     var authenticatedRolePolicy = Aws.Iam.GetPolicyDocument.Invoke(new()
+    ///     var authenticatedRolePolicyPolicyDocument = Aws.Iam.GetPolicyDocument.Invoke(new()
```

Comment:
The last commit has only this change. Was the file hand-edited? If so, how will the change be preserved for the next SDK generation?